vnext/Microsoft.ReactNative/JsiApi.idl

```diff
@@ -65,6 +65,7 @@ namespace Microsoft.ReactNative
     Boolean,
     Number,
     Symbol,
+    BigInt,
```

Comment:
don't add new enum values in the middle as an app built against a prior version will think the value 5 is String, not BigInt

Reply:
JSI is not ABI-safe/binary compatible by design - you're always running code-gen on modules and building from source. If we stray out of sync from the enum values/order in jsi.h, everything breaks.

Reply:
(aka, all the RN unit tests broken when we put bigint at the end, rather than in the middle where they put it)